lib/vmpooler/api/v1.rb

```diff
@@ -395,7 +508,7 @@ def sync_clone_targets
           end
 
           # for backwards compatibility, include separate "empty" stats in "status" block
-          if ready == 0
+          if ready == 0 && max != 0
```

Comment:
Is that the only place where we had to check that a pool is `ondemand` based on the max value being 0?
I'd like to encapsulate that in a utility method that makes it clear the relationship between being ondemand and max ==0 in case in the future we change what criteria that is. If this is the only place then its not so bad and I can be convinced to keep it that way.

I was thinking
```suggestion
          if ready == 0 && !is_ondemand?
```

with that method checking if the pool has its max == 0. Or whatever logical inverse is if that makes sense

Reply:
I believe it shows up once in API, and once in pool manager, because we perform a similar check to avoid those messages in each location.